src/widgets/tabs.rs

```diff
@@ -143,22 +143,18 @@ impl<'a> Tabs<'a> {
     /// # Examples
     ///
     /// A space on either side of the tabs.
-    ///
-    /// eg " Tab1 │ Tab2 │ Tab3 │ Tab4 "
     /// ```
     /// # use ratatui::{prelude::*, widgets::Tabs, symbols};
-    /// let tabs = Tabs::new(vec!["Tab 1", "Tab 2"]).padding(1, 1);
+    /// let tabs = Tabs::new(vec!["Tab 1", "Tab 2"]).padding(" ", " ");
     /// ```
-    /// No spaces on either side of the tabs.
-    ///
-    /// eg "Tab1│Tab2│Tab3│Tab4"
+    /// Nothing on either side of the tabs.
     /// ```
     /// # use ratatui::{prelude::*, widgets::Tabs, symbols};
-    /// let tabs = Tabs::new(vec!["Tab 1", "Tab 2"]).padding(0, 0);
+    /// let tabs = Tabs::new(vec!["Tab 1", "Tab 2"]).padding("", "");
     /// ```
-    pub fn padding(mut self, left: u16, right: u16) -> Tabs<'a> {
-        self.padding_left = left;
-        self.padding_right = right;
+    pub fn padding<T>(mut self, left: T, right: T) -> Tabs<'a> where T: Into<Line<'a>> {
```

Comment:
Looking at both left and right having to be the same `T` here, I think this would work better as two methods rather than one. This won't otherwise allow `padding("a", "b".red())`

Reply:
Oh true, I am away for a few days but I will correct that once I am back.